# Array of obstacles
std_msgs/Header header
ObstacleWpnt[] obstacles
